Source/Android/jni/MainAndroid.cpp

```diff
@@ -433,6 +433,8 @@ JNIEXPORT jint JNICALL Java_org_dolphinemu_dolphinemu_NativeLibrary_GetPlatform(
                                                                                 jstring jFilename);
 JNIEXPORT jstring JNICALL
 Java_org_dolphinemu_dolphinemu_NativeLibrary_GetVersionString(JNIEnv* env, jobject obj);
+JNIEXPORT jstring JNICALL Java_org_dolphinemu_dolphinemu_NativeLibrary_GetGitRevision(JNIEnv* env,
```

Comment:
if you can use the BuildConfig.VERSION_NAME instead of a native function it would be a lot simpler.  We use it in now to show the version in the tittlebar. It's "5.0-5630-debug" for me now.

Reply:
That has the disadvantage of not distinguishing between versions with the same version name but different git hashes (for instance, if you switch between two PRs that both have one commit and are based on the same version of master). And it wouldn't be that much simpler, it would just save us from having one extra JNI function.

Reply:
Ah, that is true. If you want the git hash, it could be added to BuildConfig via buildConfigField(). It feels right to have all the build info in the same BuildConfig object. I think that's cleaner than JNI.